ui/jobs/components/drk.ts

```diff
@@ -4,9 +4,9 @@ import { kAbility } from '../constants';
 import { computeBackgroundColorFrom } from '../utils';
 
 let resetFunc: ((bars: Bars) => void) = (_bars: Bars) => undefined;
-let tid1: ReturnType<typeof setTimeout>;
-let tid2: ReturnType<typeof setTimeout>;
-let tid3: ReturnType<typeof setTimeout>;
+let tid1: number;
```

Comment:
I had this same conversation several times with @Trim21 in https://github.com/quisquous/cactbot/pull/3001#discussion_r647138823 (and earlier in that PR).

I agree that we are only going to run this in the browser though, so this could/should just be `number`.

Reply:
It turns out that we may need to differentiate/separate tsconfig for files run in node and browser, but are there some files run in both node and browser?

Reply:
I think my general feeling is that files are either:
* browser only (such as this file, which is dealing with html and window.setTimeout)
* node only (like utilities, which depend on fs or path or node things)
* both (which don't depend on either)

I don't know of code that we're using in node and browser that is using node/browser specifics.

Reply:
Mmmm ok I think that's not technically true.  There is stuff like `timeline.ts` that has some browser stuff in it that we use from some node files, but we don't use any of the browser code, but it does get imported.